.github/workflows/cypress.yml

```diff
@@ -54,9 +56,9 @@ jobs:
         uses: cypress-io/github-action@v4
         with:
           install: false
-          start: yarn run ci, docker run -p 9292:9292 --network=host -e DATABASE_URL=postgres://rails:password@localhost:5432 -e API_KEY=test --name forms-api forms-api
+          start: yarn run ci, docker run -p 9292:9292 --network=host -e DATABASE_URL=postgres://rails:password@localhost:5432 -e SETTINGS__FORMS_API__AUTH_KEY=$SETTINGS__FORMS_API__AUTH_KEY --name forms-api forms-api
```

Comment:
Is it this line causing issues? Cypress runs fine so I think the env variables are being past in correctly there.

This line is setting the env-var SETTINGS__FORMS_API__AUTH_KEY in the API to test. Shouldn't it be:
```
start: yarn run ci, docker run -p 9292:9292 --network=host -e DATABASE_URL=postgres://rails:password@localhost:5432 -e API_KEY=$SETTINGS__FORMS_API__AUTH_KEY --name forms-api forms-api
```

unless there are changes in the API project to use $SETTINGS__FORMS_API__AUTH_KEY instead of $API_KEY?